Bottom: 9a0bf873aaf3c5407ecee10d0dcf7cea3f3c55e9
Top:    ee3c6840d690995916b5ebe744a1921774384537
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:18:51 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index e43ec87..a8e87d5 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -417,7 +417,7 @@ ghid_note_event_location (GdkEventButton * ev)
     }
 
   /* Unproject event_x and event_y to world coordinates of the plane we are on */
-  ghid_unproject_to_z_plane (event_x, event_y, 0,
+  ghid_unproject_to_z_plane (event_x, event_y, global_depth,
                              &event_x, &event_y);
 
   gport->view_x = event_x * gport->zoom + gport->view_x0;
